### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#622] [Galaxy Trivia] [answered]
🟦◻️⭐
🟦🟨✝️
🟦🟨🟥
⬜🟧🫒🫒

**Comment by MohammedNayeem9** [comment]
Based on the emojis it looks like a country Flag🚩and my guess are
**1. Somalia 🇸🇴
2. Sweden 🇸🇪
3. Ukraine🇺🇦
4. Niger🇳🇪**

**Comment by peterbaccaro** [comment, marked as answer]
1. SOMALIA
2. SWEDEN
3. ROMANIA
4. CYPRUS

![flags4](https://user-images.githubusercontent.com/74666079/234383675-9a2d76dc-ab3b-4080-b381-0da8244ae9c0.png)

**Reply by ParanoidUser** [reply]
As always, well done! 😊
